src/app/credExplorer/repoRegistry.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+// The repoRegistry is written by the CLI load command
+// (src/cli/commands/load.js) and is read by the RepositorySelect component
+// (src/app/credExplorer/RepositorySelect.js)
+import deepEqual from "lodash.isequal";
+import {toCompat, fromCompat, type Compatible} from "../../util/compat";
+
+export const REPO_REGISTRY_FILE = "repositoryRegistry.json";
+export const REPO_REGISTRY_API = "/api/v1/data/repositoryRegistry.json";
```

Comment:
The value of this const isn’t really clear to me. Really, it seems like
`REPO_REGISTRY_API` should be `sourcecredDataUrl(REPO_REGISTRY_FILE)`,
where ``sourcecredDataUrl = (x) => `/api/v1/data/${x}` `` is used more
generally throughout the application, perhaps as part of some router.

I’m not totally sure what the right way to factor this is right now. The
current code seems okay; it would also be fine with me to delete this
export. What do you think?

Reply:
I agree that in the future we should have a `sourcecredDataUrl` method or similar and use the factorization you suggest. In the absence of that API, keeping the API const next to the FILE const is slightly nice, in that if you change the FILE const it's easy to find the corresponding API const that needs to change.